zuul.d/jobs.yaml

```diff
@@ -16,13 +16,15 @@
       - tests?\/functional
       - ^renovate.json$
     vars:
+      cifmw_install_yamls_sdk_version: v1.41.1
```

Comment:
only the changes to zuul.d/projects.yaml were required for me. I think we can drop the changes to jobs.yaml here.

Reply:
done